sale_order_team_from_product/readme/CREDITS.md

```
@@ -0,0 +1,2 @@
+The development of this module has been financially supported by:
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)